README.md

```diff
@@ -13,6 +13,7 @@ VITE_AUTHGEAR_CLIENT_ID=your Authgear app client id
 VITE_AUTHGEAR_ENDPOINT=your Authgear project URL
 VITE_AUTHGEAR_REDIRECT_URL=http://localhost:4000/auth-redirect
 ````
+Or rename `.env.example` to `.env` then add the Client ID and Endpoint for your Authgear Client Application.
```

Comment:
The convention is calling it .env.template, and not renaming it but copy the values into .env. Renaming it would mess with the version control